src/sentry/static/sentry/app/views/settings/projectDataPrivacy/projectDataPrivacyRulesPanel.tsx

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+
+import styled from '@emotion/styled';
+import {t} from 'app/locale';
+import {Panel, PanelHeader, PanelBody} from 'app/components/panels';
+import Link from 'app/components/links/link';
+import {IconAdd} from 'app/icons/iconAdd';
+
+import ProjectDataPrivacyRulesForm, {RuleType} from './projectDataPrivacyRulesForm';
+
+type State = {
+  rules: Array<RuleType>;
+  newRules: Array<React.ReactElement<ProjectDataPrivacyRulesForm>>;
+};
+
+class ProjectDataPrivacyRulesPanel extends React.Component<{}, State> {
+  state: State = {
+    rules: [],
+    newRules: [],
+  };
+
+  componentDidMount() {
+    this.handleAddNewRule();
+  }
+
+  handleDeleteNewRule = (newRuleIndex: number) => () => {
+    this.setState(prevState => ({
+      newRules: [
+        ...prevState.newRules.slice(0, newRuleIndex),
+        ...prevState.newRules.slice(newRuleIndex + 1),
+      ],
+    }));
+  };
+
+  handleAddNewRule = () => {
+    const newRulePosition = this.state.newRules.length;
+    this.setState(prevState => ({
+      newRules: [
+        ...prevState.newRules,
+        <ProjectDataPrivacyRulesForm
+          key={newRulePosition}
+          onDelete={this.handleDeleteNewRule(newRulePosition)}
+        />,
+      ],
+    }));
+  };
+
+  handleDeleteRule = (ruleId: number) => () => {
+    this.setState(prevState => ({
+      rules: prevState.rules.filter(rule => rule.id !== ruleId),
+    }));
+  };
+
+  render() {
+    const {rules, newRules} = this.state;
+    return (
+      <Panel>
+        <PanelHeader>{t('Data Privacy Rules')}</PanelHeader>
+        <PanelBody>
+          <form>
+            {rules?.map(rule => (
+              <ProjectDataPrivacyRulesForm
+                key={rule.id}
+                onDelete={this.handleDeleteRule(rule.id)}
+                {...rules}
+              />
+            ))}
+            {newRules.map(newRule => newRule)}
+          </form>
+          <PanelAction>
+            <StyledLink onClick={this.handleAddNewRule}>
+              <IconAdd circle />
+              <span>{t('Add Rule')}</span>
+            </StyledLink>
+          </PanelAction>
+        </PanelBody>
+      </Panel>
+    );
+  }
+}
+
+export default ProjectDataPrivacyRulesPanel;
+
+const PanelAction = styled('div')`
+  padding: 14px 22px;
+`;
+
+const StyledLink = styled(Link)`
```

Comment:
Also here, should we refactor to grid, theme color and padding with `space` util?